Makefile

```diff
@@ -0,0 +1,19 @@
+protoc-setup:
+	cd meshes
+	wget https://raw.githubusercontent.com/layer5io/meshery/master/meshes/meshops.proto
+
+proto:	
+	protoc -I meshes/ meshes/meshops.proto --go_out=plugins=grpc:./meshes/
+
+docker:
+	docker build -t layer5/meshery-octarine .
+
+docker-run:
+	(docker rm -f meshery-octarine) || true
+	docker run --name meshery-octarine -d \
+	-p 10000:10000 \
```

Comment:
@girishranganathan does @thehh1974 need to make use of port 10004 instead?

(istio - 10000)
(linkerd - 10001)
(consul - 10003)
(octarine -10004)
(nsm - 10005)

Reply:
Yes, definitely a nice to have to use a different port but not a mandate.